docs/source/notebooks/CaSR_basic.ipynb

```diff
@@ -0,0 +1,6506 @@
+{
```

Comment:
<p>One small suggestion :  As the link to this notebook will be provided on CaSPAR etc I think we should show that there is a <code>daily</code> and <code>hourly</code> ncml directly here</p><p> </p><p>e.g. open and display both.  Then use the daily in the rest of the notebook  </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/pavics-sdi/pull/320/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>or at least give the url for hourly even if the variable is never used?  <code>url_1hr="https://pavics.ouranos.ca/twitcher/ows/proxy/thredds/dodsC/datasets/reanalyses/1hr_RDRSv2.1_NAM.ncml"</code>  </p>

Reply:
<p>It's in the text just above Data Access</p>